app/views/common/_email_share_modal.html.erb

```diff
@@ -20,10 +20,11 @@
 
 			<div class='field'>
 				<label if-branded='color, grey'>Your Message</label>
-				<textarea type='text' name='email[message]' style="height:100px;">Join me and donate to <%= fundraiser %>. <%= fundraiser_url %></textarea>
+				<textarea type='text' name='email[message]' style="height:100px;">Join me and donate to <%= fundraiser %>.<%= "\n\n" %>Follow this link to check it out: <%= fundraiser_url %></textarea>
 			</div>
 
 			<input type='hidden' name='email[from_email]' value='<%= current_user && current_user.email %>'>
+			<input type='hidden' name='email[from_name]' value='<%= current_user && current_user.name %>'>
```

Comment:
Are all these other changes still needed?

Reply:
No, I was waiting to drop this commit once Wendy approved the change.